node/cn/filters/api.go

```diff
@@ -224,6 +224,7 @@ func RPCMarshalHeader(head *types.Header, isEnabledEthTxTypeFork bool) map[strin
 		"timestamp":        (*hexutil.Big)(head.Time),
 		"timestampFoS":     hexutil.Uint(head.TimeFoS),
 		"extraData":        hexutil.Bytes(head.Extra),
+		"size":             hexutil.Uint64(head.Size()),
```

Comment:
Reflected apis are `GetHeaderByBlock`, `GetHeaderByHash`, and subscibing NewHeads, right?

Reply:
This change can affect SDK and exiting dApp. Some SDK parses RPC returns strictly, so it fails to parse return data if it has an unexpected field. As we don't have full control of modifying all SDK including Ethereum SDK, don't change the return format if the change is not necessary.

Reply:
Reverted.